crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1159,9 +1181,49 @@ impl fmt::Debug for World {
     }
 }
 
-unsafe impl Send for World {}
 unsafe impl Sync for World {}
 
+/// A world which does not contain any [`!Send`] resources, and therefore
+/// can be safely sent between threads.
+///
+/// The name turtle is derived from this being something which is moving a
```

Comment:
🤔 I... don't hate it? Very cute. Not sure it's terribly descriptive.

What about `ThreadSafeWorld` or something?

Reply:
It is quite a niche type, to be fair.

I hoped I'd be able to sneak a bit of whimsy through.

(I will claim I didn't just create this PR for this joke; whether you believe that is your choice)

Reply:
it *is* turtles all the way down

Reply:
I have to leave a comment in support of `Turtle`s. It's personal for me, given my avatar.

Reply:
amazing